tutorials/notebooks/mct_features_notebooks/keras/example_keras_xquant.ipynb

```diff
@@ -0,0 +1,208 @@
+      ],
+      "metadata": {
+        "id": "6QnODkANmU2J"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Now we can run tensorboard:"
+      ],
+      "metadata": {
+        "id": "OAhYMxgzWF1A"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%load_ext tensorboard\n",
```

Comment:
why the "%"?

Reply:
It is used to indicate a magic command in Jupyter notebooks. Magic commands are a feature of 
 Jupyter notebooks to perform various tasks without a lot of complex code.
In this specific case, %load_ext tensorboard is a line magic command that loads the TensorBoard extension, enabling you to use TensorBoard directly within the notebook.